common/src/main/java/com/company/common/dtos/AuthResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.company.common.dtos;
+
+public class AuthResponse {
```

Comment:
Let's postpone all authorization staff, and move according to the initial plan. Security could be integrated later.

Moreover you choose spring boot 2.7 which will be not supposed starring From this November, and security was reworked significantly in latest version.